### PR TITLE
Update styles

### PR DESCRIPTION
**Pull request by sidelong44**
Added styles for radio stations
Added selector to the miniplayer

### CODE REVIEW

**Comment by sidelong44** [comment]
#11
@macfeaster Check this, plz

File: play-music.user.css [hunk 1]
```diff
@@ -172,10 +172,11 @@ sj-right-drawer #drawer.sj-right-drawer { background: #161616; }
 #playlist-drawer .autoplaylist-section, #playlist-drawer #recent-playlists-container { border-bottom: 1px solid #333; }
 #playlist-drawer .playlist-drawer-item .playlist-wrapper:hover, #playlist-drawer .playlist-drawer-item .playlist-wrapper:focus, #playlist-drawer .playlist-drawer-item sj-play-button:hover~.playlist-wrapper, #playlist-drawer .playlist-drawer-item iron-icon:hover~.playlist-wrapper, #playlist-drawer .playlist-drawer-item.playlist-drop-target:not(.subscribed) .playlist-wrapper { background: #272727; }
 #playlist-drawer:not([style-scope]):not(.style-scope) paper-header-panel[at-top]:not([style-scope]):not(.style-scope) paper-toolbar:not([style-scope]):not(.style-scope) { border-bottom: 1px solid #333; }
+paper-toolbar { background: #161616; }
 
 /* Queue */
 #queue-overlay::after { border: 8px solid #212121; }
-.sj-mini-queue-0 #queueContainer.sj-mini-queue { background: #212121; }
+sj-mini-queue #queueContainer.sj-mini-queue, .sj-mini-queue-0 #queueContainer.sj-mini-queue { background: #212121; }
```
Comment (sidelong44) [inline play-music.user.css:179]: This selector for mini queue is in my DOM.

File: play-music.user.css [hunk 2]
```diff
@@ -172,10 +172,11 @@ sj-right-drawer #drawer.sj-right-drawer { background: #161616; }
 #playlist-drawer .autoplaylist-section, #playlist-drawer #recent-playlists-container { border-bottom: 1px solid #333; }
 #playlist-drawer .playlist-drawer-item .playlist-wrapper:hover, #playlist-drawer .playlist-drawer-item .playlist-wrapper:focus, #playlist-drawer .playlist-drawer-item sj-play-button:hover~.playlist-wrapper, #playlist-drawer .playlist-drawer-item iron-icon:hover~.playlist-wrapper, #playlist-drawer .playlist-drawer-item.playlist-drop-target:not(.subscribed) .playlist-wrapper { background: #272727; }
 #playlist-drawer:not([style-scope]):not(.style-scope) paper-header-panel[at-top]:not([style-scope]):not(.style-scope) paper-toolbar:not([style-scope]):not(.style-scope) { border-bottom: 1px solid #333; }
+paper-toolbar { background: #161616; }
```
Comment (sidelong44) [inline play-music.user.css:175]: This selector set dark bg for right playlist panel

**Comment by macfeaster** [comment]
@sidelong44 Thanks for this! All of these just complement existing rules so LGTM so I merged. I guess Stylus looks at the version number so I bumped that to trigger an update. Let me know if you find anything else!

**Pull request merged**